src/CMakeLists.txt

```diff
@@ -103,3 +103,6 @@ install(TARGETS umf
 )
 
 add_subdirectory(pool)
+if(LINUX) # TODO: enable it on Windows
```

Comment:
if LINUX and UMF_BUILD_LIBUMF_POOL_JEMALLOC

Reply:
Done